spec/lib/consul_form_builder_spec.rb

```diff
@@ -28,6 +28,13 @@
       expect(page).to have_css "input[aria-describedby='dummy_title-help-text']"
     end
 
+    it "does not generate a hint attribute when provided hint without text" do
+      render builder.text_field(:title, hint: "")
+
+      expect(page).not_to have_css ".help-text"
+      expect(page).not_to have_css "input[aria-describedby='dummy_title-help-text']"
```

Comment:
In order to be consistent with the `not_to` expectation a couple of tests above this one:

```suggestion
      expect(page).not_to have_css "input[aria-describedby]"
```